### ISSUE TITLE
Should SQLite autoincrement be enabled?

### ISSUE THREAD

**Issue by benjaminmayo** [vapor/sqlite-kit#73]
https://github.com/vapor/sqlite-kit/blob/f0980810d474d092c86479c656cb50754c1e10c7/Sources/SQLiteKit/SQLiteDialect.swift#L37-L39

Is this dialect definition correct? SQLite supports autoincrement for integer primary keys, so I feel like this should be `true` and not `false`. 

I ran into this today trying to reuse a Model in a SQLite environment, and was blocked by the 'Autoincrement not supported, skipping' error path.

**Comment by calebkleveter** [comment]
Duplicate of https://github.com/vapor/sqlite-kit/issues/72.
